lib/views/help/privacy.html.erb

```diff
@@ -766,13 +766,31 @@
     <a href="#right_to_complain">#</a>
   </h3>
   <p>
-    You have the right to complain if you believe that we have mishandled your
-    personal data. For the UK, the relevant supervisory body is the Information
-    Commissioner’s Office. <a href="https://ico.org.uk/concerns/handling/">
-    You can report a concern here</a> (but do contact us
-    first, so that we can try and help — see our
-    <a href="https://www.whatdotheyknow.com/help/complaints">
-    complaints procedure here</a>).
+    You have the right to complain if you believe that we have mishandled
+    your personal data. In the UK, the relevant supervisory body is the
+    Information Commissioner’s Office (ICO). Our data protection
+    registration number is<a href="https://ico.org.uk/ESDWebPages/Entry/Z9602302">
+    ZA9602302</a>.
+  </p>
+  <p>
+    You can contact the ICO using any of the following methods:
+    <ul>
+    <li>
+      Online:&nbsp;&nbsp; <a href="https://ico.org.uk/make-a-complaint/">
+      https://ico.org.uk/make-a-complaint/</a>
+    </li>
+    <li>
+      Phone:&nbsp;&nbsp; (0303) 123 1113
+    </li>
+    <li>
+      Post:&nbsp;&nbsp; Information Commissioner’s Office, Wycliffe House,
+      Water Lane, WILMSLOW, SK9 5AF
+    </li>
+    </ul>
+    &hellip;but do <a href="<%= help_contact_path %>">contact us</a> first,
```

Comment:
I'd be inclined to put this above the ICO links.

How about:

> You have the right to complain if you believe that we have mishandled your personal data. Do contact us first, so that we can try and help (see our complaints procedure here).
>
> In the UK, the relevant supervisory body is the Information Commissioner’s Office (ICO). Our data protection registration number is ZA9602302.
>
> You can contact the ICO using any of the following methods:
> …

Reply:
Agreed - this makes sense. Pushed a fixup under 5b533a3